HLTrigger/Configuration/python/HLT_75e33/modules/hltGeneralTracks_cfi.py

```diff
@@ -24,3 +24,58 @@
     trackAlgoPriorityOrder = cms.string('trackAlgoPriorityOrder'),
     writeOnlyTrkQuals = cms.bool(False)
 )
+
+_hltGeneralTracksSingleIterPatatrack = hltGeneralTracks.clone(
+    TrackProducers = cms.VInputTag("hltInitialStepTrackSelectionHighPurity"),
+    hasSelector = cms.vint32(0),
+    indivShareFrac = cms.vdouble(1.0),
```

Comment:
```suggestion
    TrackProducers = ["hltInitialStepTrackSelectionHighPurity"],
    hasSelector = [0],
    indivShareFrac = [1.0],
```
and similar elsewhere: when cloning or modifying an existing configuration it's safer for syntax self-check to modify the parameters instead of re-defining them

Reply:
I think I fixed every instance of cloning and modifying in https://github.com/SegmentLinking/cmssw/pull/132/commits/d974b796c1fd50375abe70b96ebcfea13c522a72.